.github/workflows/main.yml

```diff
@@ -57,6 +57,9 @@ jobs:
           rm yarn.lock
           yarn && yarn build_test
 
+      - name: Check formatting of frontend files
+        run: yarn && yarn format --check --no-write
```

Comment:
shouldn't need to run `yarn` again as it's been run in the previous step

Reply:
does this github action fail loudly?

Reply:
Yep, here's a sample failure:
https://github.com/archesproject/arches/actions/runs/8652938811/job/23726977872

It adds a hint "run prettier to fix", but my plan is to broadcast pretty loudly that you can install pre-commit. Could consider adding that hint to the github output.